maven-settings/src/main/mdo/settings.mdo

```diff
@@ -621,16 +626,24 @@
           <version>1.1.0+</version>
           <type>String</type>
           <defaultValue>default</defaultValue>
-          <description>The layout of the mirror repository. Since Maven 3.</description>
+          <description>
+            <![CDATA[
+            The layout of the mirror repository.
+            <br><b>Since</b>: Maven 3
```

Comment:
Why not using `@since Maven 3` instead ? (same for the other `<br><b>Since</b>: Maven xxx` pattern)

Reply:
because I just reused the existing pattern
I don't know if `@since` works with Modello xdoc, need to check: doing the pattern change in every `.mdo` could be a good thing for another PR

Reply:
It clearly does not render very well. It would be nice if modello could be extended to understand javadoc syntax, as it's already used and is not displayed really nicely:
  https://maven.apache.org/ref/4.0.0-alpha-12/maven-settings/settings.html#proxy

Reply:
Maybe using a small library such as https://github.com/chhorz/javadoc-parser ?

Reply:
https://github.com/codehaus-plexus/modello/pull/414

Renders as 
<img width="549" alt="image" src="https://github.com/apache/maven/assets/84022/99c1d9ee-25e0-421e-b1a1-2ac493cbfd81">
 instead of 
<img width="572" alt="image" src="https://github.com/apache/maven/assets/84022/15a1a320-9b3b-4ac5-a829-d885997cf2ef">

Reply:
I'll merge this PR with the existing pattern
dropping the existing pattern to replace by better option is nice, but it's not the current objective: I'll be happy that the content is changed later